pylib/cqlshlib/test/test_cqlsh_output.py

```diff
@@ -402,7 +402,6 @@ def test_timestamp_output(self):
             """),
         ), env=env)
         try:
-            import pytz  # test only if pytz is available on PYTHONPATH
```

Comment:
@bschoening this fails the build for me locally, I had issue with that in the last patch of yours and I intentionally moved it back.

Reply:
@smiklosovic ah, ok, this has been reverted

Reply:
@bschoening  it seems like you've completely deleted that file.

Reply:
yes, accidentally, it should be reverted now

Reply:
I had been trying to clean up the last pyflake warning

% pyflakes *.py */*.py
test/test_cqlsh_output.py:405:13 'pytz' imported but unused

Reply:
@bschoening yeah I see ... weird it fails here. It is strange test. For me, when it is not there, timezones do not match and it fails. For me it is +3h from what it expects.

Reply:
It appears the name 'America/Sao_Paulo' is not in the default TZ database, so it requires a 3rd party module pytz.  PEP 615 in Python 3.9 addresses this.

PEP 615 – Support for the IANA Time Zone Database in the Standard Library

Reply:
Looks like this is designed to test the use of pytz in bin/cqlsh.py. With PEP 615, the behavior should change in Python 3.9 and later.